test/etherium_test.dart

```diff
@@ -0,0 +1,37 @@
+import 'dart:math';
+
+import 'package:flutter_test/flutter_test.dart';
+import 'package:http/http.dart';
+import 'package:wallet_core/wallet_core.dart';
+import 'package:web3dart/web3dart.dart';
+
+void main() {
+  test('Etherium test', () async {
```

Comment:
this is more of a discovery test, could we by default mark it as `skip: true` so it won't take part in CI/CD testing pipeline?

Reply:
Sounds good. Let me update that.

Reply:
Done